packages/core/src/App/Components/Layout/Header/login-button.jsx

```diff
@@ -23,7 +24,7 @@ const LoginButton = ({ className }) => {
                         location.href = 'https://hub.deriv.com/tradershub/login';
                     }
                 }
-                if (is_deriv_com) {
+                if (is_deriv_com && !is_TMB_enabled) {
```

Comment:
This might affect production, because production should still be using OIDC right, if `is_tmb_enabled` is not there then production will use Hydra 1.5 instead 👀

Reply:
Does `requestOidcAuthentication` need to be checked and wrapped with the TMB feature flag?

Reply:
the check here is when there's no `is_TMB_enabled` feature flag and is_deriv_com, then we will use OIDC. Else we're gonna use Hydra